README.md

```diff
@@ -134,6 +134,10 @@ duplicity_ng_cronjob 'myduplicity' do
   azure_account_name 'MY_ACCOUNT_NAME'
   azure_account_key  'MY_ACCOUNT_KEY'
 
+  # In case you use FTP/SFTP as your backend, your credentials go here
+  # Nb. username goes in the url like sftp://username@sftp.mycompany.com/
```

Comment:
Would you mind changing `Nb.` to `Note:`? As a non-native, I had to look up what it meant.